vhdbuilder/scripts/automate_generate_version.sh

```diff
@@ -4,22 +4,23 @@ set -euxo pipefail
 build_ids=$1
 global_image_version="${IMAGE_VERSION:=}"
 for build_id in $build_ids; do
-    for artifact in $(az pipelines runs artifact list --run-id $build_id | jq -r '.[].name'); do    # Retrieve what artifacts were published
+    artifacts=($(az pipelines runs artifact list --run-id $build_id | jq -r '.[].name'))
+    artifacts=(${artifacts[@]//vhd*})
+    printf '%s\n' "${artifacts[@]}"
+    for artifact in ${artifacts[@]}; do    # Retrieve what artifacts were published
         # This loop is because of how the Image Version is set for builds. 
         # It uses the UTC time of when the build for a particular SKU ends. 
         # So in the past, it has happened that you trigger a build at say 3/4pm PST, 
         # some SKUs will have todays date some will have tomorrows based on when they are triggered because of UTC conversion
         # TODO(amaheshwari): Change VHD script to use a common var for image version that is plumbed down to all SKUs
-        if [[ $artifact == *"publishing-info"* ]]; then
```

Comment:
might be worth keeping this check for sanity's sake? 

I doubt the artifact names are gonna change anytime soon but would still be nice to have I think, up to you though

Reply:
fair enough, since we already got some gains on speed, I will keep this check as is.

Reply:
resolved